surprise/similarities.pyx

```diff
@@ -359,3 +362,112 @@ def pearson_baseline(n_x, yr, min_support, global_mean, x_biases, y_biases,
             sim[xj, xi] = sim[xi, xj]
 
     return sim
+
+
+def spearman(n_x, yr, min_support):
+    """Compute the Spearman correlation coefficient between all pairs of users
+    (or items).
+
+    Only **common** users (or items) are taken into account. The Spearman
+    correlation coefficient can be seen as a non parametric Pearson's
```

Comment:
What do you mean non-parametric?

I'd like to add something like "The spearman correlation coefficient is equivalent to Pearson correlation coefficient where the ratings are replaced by their rankings."

Reply:
Okay I've improved on that.